### DISCUSSION TITLE
Multi Subnet question

### DISCUSSION THREAD

**Discussion by FrontWhite** [strongswan/strongswan#1699] [Q&A] [answered]
I used swanctl.conf to configue，and I have two questions：
1. Whether IKEv1 or IKEv2 support Mixed ipv4 and ipv6 subnet segments, such as:
local_ts:192.168.1.0/24,2001::2/128,192.168.2.0/24 ?
2. if use ikev1,  how to configue multiple subnet segments,can you give me an example？

**Comment by tobiasbrunner** [comment, marked as answer]
> 1. Whether IKEv1 or IKEv2 support Mixed ipv4 and ipv6 subnet segments, such as:
>        local_ts:192.168.1.0/24,2001::2/128,192.168.2.0/24 ?

IKEv2 yes (unless you connect to a peer that doesn't support it, like Cisco), IKEv1 no (because it only support a single traffic selector per CHILD_SA).



> 2\. if use ikev1,  how to configue multiple subnet segments,can you give me an example？

You have to configure separate child sections for each combination of local and remote subnet.
